packages/flutter/lib/src/material/switch.dart

```diff
@@ -89,6 +92,37 @@ class Switch extends StatefulWidget {
   /// Defaults to accent color of the current [Theme].
   final Color activeColor;
 
+  /// The color to use on the track when this switch is on.
+  ///
+  /// Defaults to accent color of the current [Theme] withAlpha(0x80).
+  final Color activeTrackColor;
+
+  /// The color to use on the thumb when this switch is off.
+  ///
+  /// Defaults to:
+  /// if the brightness of current [Theme] is dark and [onChanged] callback is
+  /// not null: grey with shade400;
+  /// if the brightness of current [Theme] is dark and [onChanged] callback is
+  /// null: grey with shade800;
+  /// if the brightness of current [Theme] is not dark and [onChanged] callback
+  /// is not null: grey with shade50;
+  /// if the brightness of current [Theme] is not dark and [onChanged] callback
+  /// is null: grey with shade400.
+  final Color inactiveThumbColor;
+
+  /// The color to use on the track when this switch is off.
+  ///
+  /// Defaults to:
+  /// if the brightness of current [Theme] is dark and [onChanged] callback is
+  /// not null: white30;
+  /// if the brightness of current [Theme] is dark and [onChanged] callback is
+  /// null: white10;
+  /// if the brightness of current [Theme] is not dark and [onChanged] callback
+  /// is not null: black26;
+  /// if the brightness of current [Theme] is not dark and [onChanged] callback
+  /// is null: black12.
+  final Color inactiveTrackColor;
```

Comment:
these two comments will each end up all jumbled into one paragraph.

We should probably just say "Defaults to the colors described in the Material design specification" or some such. I doubt readers are going to care about exactly what shade of white it happens to default to.